src/components/editableField/EditableField.js

```diff
@@ -44,12 +55,9 @@ export const EditableField = ({ type = 'text', val = '', editState = false }) =>
 
   return (
     <div className="editablefield">
-      { isHidden ? whatInput(type) : (
-        <span
-          onClick={changeHidden}
-        >{value}
-        </span>
-      )
+      { isHidden
+        ? whatInput(type)
```

Comment:
You don't need to pass `type` since it is an accessible variable inside the `EditableField` function

Reply:
Also do not use textarea like that:
```
...>{value}
    </textarea>
```
You've already passed the value through the props